Backpacks.cs

```diff
@@ -581,6 +591,63 @@ private bool VerifyCanOpenBackpack(BasePlayer looter, ulong ownerId)
             return true;
         }
 
+        void OnPlayerConnected(BasePlayer player)
+        {
+            CreateGUI(player);
+        }
+
+        void OnPlayerSleepEnded(BasePlayer player)
+        {
+            CreateGUI(player);
+        }
+
+        void CreateGUI(BasePlayer player)
+        {
+            if (player == null || (player is NPCPlayer) || (player is HTNPlayer))
+                return;
+
+            if (!permission.UserHasPermission(player.UserIDString, GUIPermission))
+                return;
```

Comment:
We should probably check that the player has both the `use` and `gui` permissions in case they have a stale `gui` perm but no `use` perm since they would be able to see the button but opening it would do nothing.

Reply:
I'm not agree. It's up to admin to give one and not giving another, for example as advertising payed `use` with `gui` for free.

Reply:
`but opening it would do nothing` it's not true. Pressing the button will run a command just like expected and if there will be no `use` permission player will have "No Permission" message printed.

Reply:
LaserHydra can decide.